zqd/storage/archivestore/archivestore.go

```diff
@@ -22,7 +23,13 @@ func Load(path string, cfg *storage.ArchiveConfig) (*Storage, error) {
 	if cfg != nil && cfg.OpenOptions != nil {
 		oo.LogFilter = cfg.OpenOptions.LogFilter
 	}
-	ark, err := archive.CreateOrOpenArchive(path, co, oo)
+
+	arkroot, err := iosrc.ParseURI(path)
+	if err != nil {
+		return nil, fmt.Errorf("invalid archive path: %s", path)
+	}
+
+	ark, err := archive.CreateOrOpenArchive(arkroot, co, oo)
```

Comment:
I know system test coverage isn't extensive for zqd/s3 but would be nice to include one for a space with an s3 datapath.